tufte/components/summarizer.py

```diff
@@ -36,12 +36,12 @@ def _get_column_properties(self, df: pd.DataFrame, n_samples: int = 3) -> List[D
 
         def add_date_properties(column):
             try:
-                properties["min"] = df[column].min()
-                properties["max"] = df[column].max()
+                properties["min"] = json.dumps(df[column].min().timestamp())
+                properties["max"] = json.dumps(df[column].max().timestamp())
```

Comment:
Use .isoformat() which is human readable. timestamp() is a number that cannot be read as a date